app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Http\Requests\RegisterRequest;
+use Illuminate\Support\Facades\Storage;
+use App\Http\Resources\UserResource;
+use Validator;
+
+use App\Models\User;
+use App\Services\UserService;
+
+class UserController extends Controller
+{
+    protected $userService;
+
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    /**
+     * Register
+     *
+     * @param  [string] name
+     * @param  [string] username
+     * @param  [string] email
+     * @param  [string] password
+     * @param  [file] avatar
+     * @param  [string] password_confirmation
+     * @return [json] message ft record
+     */
+
+    public function register(RegisterRequest $request)
```

Comment:
Comment này có cần không e, nếu viết đúng PHP docblock thì chỉ có:
```suggestion
    /**
     * Register
     *
     * @param RegisterRequest $request
     * @return UserResource
     */
    public function register(RegisterRequest $request)
```

Cơ mà mình dùng typehint là `RegisterRequest $request` rồi thì ko cần viết cái docblock comment này cũng được

Reply:
Dạ vâng để em xóa nó đi ạ 👍